### ISSUE TITLE
Support wrong reprcodes

### ISSUE THREAD

**Issue by achaikou** [equinor/dlisio#219]
Situation:
Template: reprcode is 0 for one attribute (which we support in hope that it would be overwritten by objects).
Objects: in majority of objects code is overwritten with something useful, but in some default value is used. This makes attribute/object spoiled, but that shouldn't prevent us from reading all other objects and rest of the file.
Now we hard-fail with _"unable to patch attribute with no value: unknown representation code 66"_
Around 100 files found

**Comment by achaikou** [comment]
For now we have 3 tests that attempt to define behavior in similar cases

`test_logical_record/test_repcode_invalid_in_template_value`
   template has invalid reprcode and some value; reprocode and value are correctly overriden in object -> _unknown representation code error_
`test_logical_record/test_repcode_invalid_in_template_no_value`
   template has invalid reprcode and no value; reprcode and value are correctly overriden on object -> _correct load_
`test_logical_record/test_repcode_invalid_in_objects`
   template is correct; value and reprcode are incorrectly overriden in object -> _unknown representation code error_

Situation presented in the files seem to be the following:
   template has invalid reprcode and no value; nothing is overriden in object -> _unknown representation code error_
(Attribute in template has reprcode 0 and no value; attribute in object is left as default)

Test structure would be:
```
        'data/chap3/start.dlis.part',
        'data/chap3/template/invalid-repcode-no-value.dlis.part',
        'data/chap3/object/object.dlis.part',
        'data/chap3/objattr/empty.dlis.part',
```

Apart from fixing issue in general (do not allow error in one attribute to spoil the read of the whole set) we would need to define whether we can assume this situation to be somewhat correct and, for example, just mark this attribute as absent or supply default value.

**Comment by ErlendHaa** [comment]
resolved by #296
